include/psa/crypto_values.h

```diff
@@ -447,6 +455,12 @@
                         ((type) & PSA_KEY_TYPE_ECC_CURVE_MASK) : \
                         0))
 
+/**@}*/
+
+/** \defgroup iana_groups TLS Supported Groups
```

Comment:
Because the documentation of algorithm and key identifiers is currently a big swirling mass without structure. This was an attempt to co-locate the type definition - with the documentation about other ECC values - and the defined values of that type.

A better alternative might be to provide more structure by splitting the single keys & algorithm bucket into multiple sections divided by key and algorithm types. This makes it make easier to reference the relevant algorithms from the documentation on the related functions. I'll revert this change and revisit it once #149 (migrate to reST) is done.